src/includes/class-health-check-htaccess-viewer.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Class for displaying the .htaccess file for the site.
+ *
+ * @package Health Check
+ */
+
+/**
+ * Class Health_Check_Htaccess
+ */
+class Health_Check_Htaccess_Viewer {
+	/**
+	 * Callback function for the tool_tabs filter that adds content
+	 * to the tools accordion.
+	 *
+	 * @param  array $tabs Associative array of tools.
+	 *
+	 * @return array Associative array.
+	 */
+	public static function tools_tab( $tabs ) {
+		// check if it is a file.
+		if ( ! is_file( ABSPATH . '.htaccess' ) ) {
+			return $tabs;
+		}
+
+		// It does so let's get the contents
+		$htaccess_file = file_get_contents( ABSPATH . '/.htaccess' );
```

Comment:
I would add a check that we can actually read the file contents, as in some scenarios, it is "locked" from the web user.

Reply:
true, will look into this either today or tomorrow.